src/Network/Network/RouteServer/NewAzureRMRouteServerCommand.cs

```diff
@@ -50,6 +50,12 @@ public partial class NewAzureRmRouteServer : RouteServerBaseCmdlet
         [ValidateNotNullOrEmpty]
         public string HostedSubnet { get; set; }
 
+        [Parameter(
+            Mandatory = true,
```

Comment:
adding a new mandatory parameter will cause a breaking change.
Where is your design review?

Reply:
https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/969

Reply:
Please review. Thanks